shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -268,7 +268,12 @@ private void setSystemChromeEnabledSystemUIMode(PlatformChannel.SystemUiMode sys
 
     if (systemUiMode == PlatformChannel.SystemUiMode.LEAN_BACK) {
       // LEAN BACK
-      // Available starting at SDK 16
+      // Available starting at SDK 16.
+      //
+      // If the Flutter app targets SDK 15 (Flutter does this by default), then this mode will
```

Comment:
This is super confusing because this comment is mixing the public name, android 15, with the developer focused api level which is 35. Additionally flutter does target api 35 by default *today* but wont 2 years and this documentation will be incorrect. 

Here and below. 

```suggestion
      // If the Flutter android app targets api 35 or later then the android system will ignore this value unless the app also follows the opt out instructions found in https://docs.flutter.dev/release/breaking-changes/default-systemuimode-edge-to-edge.
```